### PR TITLE
Siltbreaker Tooltips/ Rename Vampire Tooth to Vampire Fang/ Fix custom hero ability tooltips 

### PR DESCRIPTION
**Pull request by OTAJumperjay**
would have been very funny if he actually channeled beans though xD

### CODE REVIEW

File: game/resource/English/ability/units/siltbreaker/tooltip_spider_boss.txt [hunk 1]
```diff
@@ -3,27 +3,26 @@
...
 
 "dota_tooltip_ability_spider_boss_passive"                        "Ankaboot passive"
-"dota_tooltip_ability_spider_boss_passive_description"            "Ankaboot presence"
+"dota_tooltip_ability_spider_boss_passive_description"            "When you get attacked by the Spiderboss, then you'll be able to see her - no matter if you have vision or not."
+"dota_tooltip_ability_spider_boss_passive_lore"                   "Mother's watching you (even through walls)"
 
 "dota_tooltip_ability_spider_boss_rage"                           "Ankaboot rage"
-"dota_tooltip_ability_spider_boss_rage_description"               "Ankaboot becomes filled with rage dealing more damage and great lifesteal"
+"dota_tooltip_ability_spider_boss_rage_description"               "Ankaboot becomes filled with rage, gaining %bonus_damage% bonus damage, %bonus_movespeed_pct%%% additional Movementspeed and %lifesteal_pct%%% lifesteal for %duration% seconds.
```
Comment (imaGecko) [inline game/resource/English/ability/units/siltbreaker/tooltip_spider_boss.txt]: missing closing "

**Comment by OTAJumperjay** [comment]
Haven't looked at it after commit .. well there is still more stuff that i'll add to this pr ... and imaGecko please don't steal all the work again haha

**Comment by OTAJumperjay** [comment]
okay siltbreaker nonsense is done... i still have to change pl and furion to have their custom tooltips

**Comment by chrisinajar** [comment]
i'm waiting for the "WIP" part to disappear before reviewing this. just reminding you it's there.

File: game/scripts/npc/abilities/siltbreaker/lycan_boss_summon_wolves.txt [hunk 2]
```diff
@@ -26,6 +26,15 @@
         //-------------------------------------------------------------------------------------------------------------
         "AbilityManaCost"               "20"
 
+        // Stats
```
Comment (chrisinajar) [inline game/scripts/npc/abilities/siltbreaker/lycan_boss_summon_wolves.txt]: these need to be in the 
```
         "AbilitySpecial"
         {
             "01"
             {
.....
```

`GetSpecialValueFor` returns the AbilitySpecial value with that name at the current level.

Comment (OTAJumperjay) [reply game/scripts/npc/abilities/siltbreaker/lycan_boss_summon_wolves.txt]: okay i'll fix that later ... roleplaying right now and i'll still have to do furion/phantom lancer custom tooltips

File: game/resource/English/ability/units/siltbreaker/tooltip_lycan_boss.txt [hunk 3]
```diff
@@ -3,23 +3,23 @@
 //=============================================================================
 
 
-"DOTA_Tooltip_Ability_lycan_boss_summon_wolves"                               "Summon Wolves"
-"DOTA_Tooltip_Ability_lycan_boss_summon_wolves_Description"                   "LongClaw howl to summon a powerfull pack of wolf beasts."
+"DOTA_Tooltip_Ability_lycan_boss_summon_wolves"								"Summon Wolves"
+"DOTA_Tooltip_Ability_lycan_boss_summon_wolves_Description"					"LongClaw howls. Summoning %num_hound_spawn% hounds, %num_hound_boss_spawn% boss hounds and %num_werewolf_spawn% werewolfs. When LongClaw used shapeshift he'll summon %num_hound_ss_spawn% hounds %num_hound_boss_ss_spawn% boss hounds and %num_werewolf_ss_spawn% werewolfs."
```
Comment (imaGecko) [inline game/resource/English/ability/units/siltbreaker/tooltip_lycan_boss.txt]: Abilities don't have their stats as part of the description

File: game/resource/English/ability/units/siltbreaker/tooltip_ogre_seer_area_ignite.txt [hunk 4]
```diff
@@ -2,8 +2,8 @@
 // Ogre Seer Ignite
 //=============================================================================
 "DOTA_Tooltip_Ability_ogre_seer_area_ignite"                              "Ignite"
-"DOTA_Tooltip_Ability_ogre_seer_area_ignite_Description"                  "Ogre Seer launches into a target area dealing damage and slowing movement speed."
-"DOTA_Tooltip_Ability_ogre_seer_area_ignite_burn_damage"                  "DAMAGE:"
-"DOTA_Tooltip_Ability_ogre_seer_area_ignite_slow_movement_speed_pct"      "SLOW:"
-"DOTA_Tooltip_Ability_ogre_seer_area_ignite_area_duration"                "DURATION:"
-"DOTA_Tooltip_Ability_ogre_seer_area_ignite_radius"                       "Radius:"
+"DOTA_Tooltip_Ability_ogre_seer_area_ignite_Description"                  "Ogre Seer launches a pit of fire into a target area, dealing %burn_damage% damage per second and slows for %slow_movement_speed_pct%%% movement speed in a %radius% radius. The pit lasts for %area_duration% seconds and lingers for %duration% seconds on the hero after leaving the pit."
```
Comment (imaGecko) [inline game/resource/English/ability/units/siltbreaker/tooltip_ogre_seer_area_ignite.txt]: changed it from correct to wrong here

**Comment by imaGecko** [comment]
Item Formatting:
Separate passive stats (strength, hp), item ability stats as part of the description (aura radius, bash duration)

Ability Formatting:
Short description of ability function, separate ability stats

**Comment by OTAJumperjay** [comment]
Let's say in the kv file i had a key called "bash_duration" for example then i write "Dota_Tooltip_Ability_filename_bash_duration"    "BASH DURATION:" or "+$bash_duration"? I know you told me several times but i am still never sure

**Comment by imaGecko** [comment]
"+$something" is only for items
if you look under files changed in the fireblast, it was correct before you changed it, serves as a good example as well

**Comment by OTAJumperjay** [comment]
Okay will fix it when i'm back home today in like 15 hours 👍🏿

**Comment by OTAJumperjay** [comment]
gonna close this one because i wanna try if travis likes me more if i re-PR this

**Pull request closed without merging**